### PR TITLE
[AzureMonitorExporter] prepare new release. Exporter 1.0.0. Distro beta7

### PR DESCRIPTION
**Pull request by TimothyMothra**
preparing new release

- AzureMonitorExporter -> v1.0.0 🎉
- AzureMonitorDistro -> v1.0.0-beta.7

### CODE REVIEW

**Review by vishweshbankwar — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:3]: Update the date

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md]: changed to today's date.

**Comment by rajkumar-rangaraj** [comment]
I recommend including distro bump here, it will help them group and also no need to wait for multiple CI runs.

**Review by rajkumar-rangaraj — APPROVED** [review]
LGTM, merge the changelog updates for distro before the merge.

**Pull request merged**
